archimedes/src/app/component/image/image.component.html

```diff
@@ -0,0 +1,67 @@
+<div *ngFor="let image of images" class="col-sm-6 col-md-4 col-xl-3">
+    <div class="card mb-3">
+        <a href="{{image.download_url}}" target="_blank">
+            <img class="card-img-top" src="{{image.thumbnail_url}}" alt="{{image.location}}">
+        </a>
+        <div class="card-block">
+
+            <p class="card-text" (click)="(selectedImage === image.id) ? selectedImage = 0 : selectedImage = image.id">
```

Comment:
The "view image details" toggle element should be a button, or maybe a link (or! a button styled as a link, with `btn-link`?).
Ideally it would also utilize `aria-expanded` and `aria-controls` (see Athena collapse examples)